tests/E2E/test/campaigns/common_scenarios/category.js

```diff
@@ -113,17 +114,20 @@ module.exports = {
       test('should search for category ', () => client.searchByValue(CategorySubMenu.search_input, CategorySubMenu.search_button, categoryData.name + date_time));
       test('should click on "Edit" action', () => {
         return promise
-          .then(() => client.clickOnAction(CategorySubMenu.update_button))
-          .then(() => client.getParamFromURL('id_category', 2000));
+          .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.update_button))
+          .then(() => client.getURL(2000))
```

Comment:
why 2000?

Reply:
2s is the time used before to wait loading page after click on reset.
We can't use wait for selector to be visible, we should wait for navigation to finish (and we can do it with selenium)

Reply:
Replace it with regex